src/applications/coronavirus-chatbot/createCoronavirusChatbot.js

```diff
@@ -1,6 +1,9 @@
 import recordEvent from 'platform/monitoring/record-event';
 import { GA_PREFIX } from './utils';
 import * as Sentry from '@sentry/browser';
+import React from 'react';
```

Comment:
does `React` have to be imported here? it seems like only `ReactDOM` is being used

Reply:
`React` is still needed for including the `ChatbotLoadError` JSX component here